irc/history/history.go

```diff
@@ -22,25 +22,52 @@ const (
 	Quit
 	Mode
 	Tagmsg
+	Nick
 )
 
+// a Tagmsg that consists entirely of junk tags is not stored
+var junkTags = map[string]bool{
```

Comment:
may be good to rename this to transientTags, it's a bit more accurate and feels a lil less critical~